src/libs/barcode/lv_barcode.h

```diff
@@ -81,7 +81,7 @@ void lv_barcode_set_direction(lv_obj_t * obj, lv_dir_t direction);
 /**
  * Set the data of a barcode object
  * @param obj pointer to barcode object
- * @param data data to display
+ * @param data to display
```

Comment:
I'll drop this... it's `@param data` + `data to display` not a typo. I have a rule to avoid this, but sometimes other things trip me up